napari/layers/image/_image_slice.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from ._image_view import ImageView
+from ...types import ArrayLike, ImageConverter
+
+
+class ImageSlice:
+    """The slice of the image that we are currently viewing.
+
+    Right now this just holds the image and its thumbnail, however future async
+    and multiscale-async changes will likely grow this class a lot.
+
+    Parameters
+    ----------
+    view_image : ArrayLike
+        The default image for the time and its thumbail.
+    image_converter : ImageConverter, optional
+        ImageView uses this to convert from raw to viewable.
+
+    Attributes
+    ----------
+    image : ImageView
+        The main image for this slice.
+
+    thumbnail : ImageView
+        The smaller thumbnail image for this slice.
```

Comment:
@pwinston just in case you were wondering about what this "thumbnail" actually is, it's not actually the thumbnail itself, but the sliced source data for the thumbnail, which for a normal image is just the image itself, but for a multiscale is the `_thumbnail_level` level of the pyramid.

Apologies that it is a bit weird, but hope that helps in case it was unclear. Probably when all is said and done we can figure out a better way to do this

Reply:
Yeah I kind of suspected it was not really the thumbnail. One of the old variables was `thumbnail_source` and I probably shouldn't have shortened it. We should clear it up either just in docs or change the var names back. I'll look at in the next PR.